test_generators/epoch_processing/main.py

```diff
@@ -14,7 +15,8 @@ def create_suite(transition_name: str, config_name: str, get_cases: Callable[[],
         -> Callable[[str], gen_typing.TestSuiteOutput]:
     def suite_definition(configs_path: str) -> gen_typing.TestSuiteOutput:
         presets = loader.load_presets(configs_path, config_name)
-        spec.apply_constants_preset(presets)
+        spec_phase0.apply_constants_preset(presets)
+        spec_phase1.apply_constants_preset(presets)
```

Comment:
Does this generator actually use phase1 at all?

Reply:
So the reason for this is what @hwwhww  pointed out below in her comment about the decorators. Because of the way they are constructed, the the test generators now run against phase 0 and phase 1. That means that without this the state generator is broken. (i.e. running `make eth2.0-spec-tests/tests/sanity` fails)